calculators/memoria_rating.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+$GLOBALS['rating_calculators']['memoria_rating'] = [
```

Comment:
please commit only one rating at a time. Remove this one, let us only focus on the _memoria_rating in this pull request.